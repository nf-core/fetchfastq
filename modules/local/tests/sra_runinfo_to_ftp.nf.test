nextflow_process {

    name "Test Process SRA_RUNINFO_TO_FTP"
    script "modules/local/sra_runinfo_to_ftp.nf"
    process "SRA_RUNINFO_TO_FTP"
    tag "sra_runinfo_to_ftp"
    tag "modules_local"

    test("Should run without failures") {

        when {
            params {
                outdir   = "$outputDir"
            }
            process {
                """
                input[0] = file("$baseDir/modules/local/tests/input/sra_runinfo_to_ftp/SRR13191702.runinfo.tsv", checkIfExists: true)
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() },
            )
        }

    }

}
