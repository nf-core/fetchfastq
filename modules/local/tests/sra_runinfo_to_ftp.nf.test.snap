{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "SRR13191702.runinfo_ftp.tsv:md5,94378c448c044b3e20e5c54e442ab62e"
                ],
                "1": [
                    "versions.yml:md5,e95f8185f665127a73622a19d321bcca"
                ],
                "tsv": [
                    "SRR13191702.runinfo_ftp.tsv:md5,94378c448c044b3e20e5c54e442ab62e"
                ],
                "versions": [
                    "versions.yml:md5,e95f8185f665127a73622a19d321bcca"
                ]
            }
        ],
        "timestamp": "2023-07-06T02:05:52+0000"
    }
}